.travis.yml

```diff
@@ -4,6 +4,12 @@ node_js:
   - 8.11.3
 env:
   - NODE_ENV="test"
+services:
+  - postgresql
+before_script:
+  - yarn db:setup:user
+  - yarn db:reset
 script:
   - yarn test
-  - yarn lint
+  - yarn enforce
+  - yarn lint
```

Comment:
nit: could you move the `yarn lint` to before the `yarn test`? this doesn't really matter for this specific project, but it's a good practice b/c some repos which contain thousands of test suites may take, say, 15+ minutes to run. Catching a potentially minor linting error at the end would necessitate fixing that error, deploying that change, and re-running the entire testing suite for another 15+ minutes. The time it takes to run the linter would comparatively be much faster than it takes to run the test, so If we caught the linting error at the beginning that would def be more worthwhile as we prefer to fail fast

Reply:
That makes sense, I pushed a new change to resolve this!